CMakeLists.txt

```diff
@@ -186,6 +186,47 @@ endif(ACF_SERIALIZE_WITH_CEREAL)
 
 message("ACF_ADD_TO_STRING : ${ACF_ADD_TO_STRING}")
 
+################
+## TEST DATA ###
+################
+
+# Only download test data if tests are enabled and we are running 
+# a recent enough version of Hunter to support hunter_private_data()
+if(ACF_BUILD_TESTS AND COMMAND hunter_private_data)
```

Comment:
`if(... AND COMMAND hunter_private_data)` would it be better to return an error and ask for update?

Reply:
I've added a `FATAL_ERROR` in this case with a message telling the user to update Hunter.